package/gluon-status-page-api/files/lib/gluon/status-page/www/cgi-bin/dyn/neighbours-nodeinfo

```diff
@@ -8,6 +8,7 @@ badrequest() {
 
 echo 'Access-Control-Allow-Origin: *'
 
-batctl if | cut -d: -f1 | grep -qxF "$QUERY_STRING" || badrequest
+#batctl if | cut -d: -f1 | grep -qxF "$QUERY_STRING" || badrequest
```

Comment:
why leaving the old commands there? if we need them again, we could look them up in git history.
(my comment also applies to the other file changes in your commit)